x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageIT.java

```diff
@@ -178,7 +178,7 @@ public void testScaleFromEmptyLegacy() {
                     .put(INDEX_STORE_STATS_REFRESH_INTERVAL_SETTING.getKey(), "0ms")
                     .put(IndexMetadata.INDEX_ROUTING_INCLUDE_GROUP_SETTING.getKey() + "data_tier", "hot")
                     .build()
-            )
+            ).setWaitForActiveShards(ActiveShardCount.NONE)
```

Comment:
This was waiting 30 seconds for a shards that could not be allocated. Following call updates settings for the correct allocation anyways...